python/pyarrow/__init__.py

```diff
@@ -212,6 +212,7 @@ def _has_pkg_config(pkgname):
                                 '--exists', pkgname]) == 0
     except OSError:
         # TODO: replace with FileNotFoundError once we ditch 2.7
```

Comment:
I didn't understand exactly what this comment was proposing (replace L213 or L215?) but surely someone else does.

Reply:
I understand the comment as that you can replace OSError on L213 with FileNotFoundError 
(FileNotFoundError didn't yet exist on Python 2.7)

Reply:
Cool thanks, will do